server/tso/allocator_manager.go

```diff
@@ -336,6 +336,9 @@ func (am *AllocatorManager) updateAllocator(ag *allocatorGroup) {
 func (am *AllocatorManager) HandleTSORequest(dcLocation string, count uint32) (pdpb.Timestamp, error) {
 	am.RLock()
 	defer am.RUnlock()
+	if len(dcLocation) == 0 {
+		dcLocation = config.GlobalDCLocation
+	}
```

Comment:
Why set dcLocation as global instead of throwing error here?

Reply:
Considering the forward compatibility, I add this check in order to handle the old TSO request that may not have this field.